build.sbt

```diff
@@ -7,13 +7,13 @@ import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
 lazy val scalaCheckVersion = "1.13.4"
 lazy val scalaTestVersion = "3.0.1"
 lazy val disciplineVersion = "0.7.2"
-lazy val catsVersion = "0.9.0"
+lazy val catsVersion = "1.0.0-MF"
 lazy val catalystsVersion = "0.0.5"
 
 lazy val buildSettings = Seq(
   organization := "org.typelevel",
   scalaVersion := "2.12.1",
-  crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.1")
+  crossScalaVersions := Seq("2.10.6", "2.11.11", "2.12.1")
```

Comment:
can we change to `2.12.3` here?

Reply:
So the dilemma is that these require scala-js changes which require changes to your `build.sbt`. 

I didn't want to do that on the original PR in case you wanted to have a minimal dependency shift, however I can get these done.

Reply:
Apparently so did, `2.11.11` so it needed to be done either way.